lib/kitchen/directions/bake_references/main.rb

```diff
@@ -6,17 +6,49 @@ module Directions
     #
     module BakeReferences
       def self.v1(book:, metadata_source:, numbered_title: false)
-        V1.new.bake(
+        V1.new.bake(book: book, numbered_title: numbered_title)
+
+        chapter_area_references = book.chapters.search('.os-chapter-area').cut
+
+        section(
           book: book,
           metadata_source: metadata_source,
-          numbered_title: numbered_title
+          chapter_area_references: chapter_area_references.paste,
+          plural: false
         )
       end
 
       def self.v2(book:, metadata_source:)
-        V2.new.bake(
+        V2.new.bake(book: book)
+
+        chapter_area_references = book.chapters.search('.os-chapter-area').cut
+
+        section(
+          book: book,
+          metadata_source: metadata_source,
+          chapter_area_references: chapter_area_references.paste
+        )
+      end
+
+      def self.v3(book:, metadata_source:)
+        V3.new.bake(book: book)
+
+        chapter_area_references = book.chapters.references.cut
+
+        section(
           book: book,
-          metadata_source: metadata_source
+          metadata_source: metadata_source,
+          chapter_area_references: chapter_area_references.paste
+        )
+      end
+
+      def self.section(book:, metadata_source:, chapter_area_references:, plural: true)
```

Comment:
actually, this method doesn't seem to do much but wrap the call to `CompositePageContainer`; personally, I don't think it's worth the few lines of code it saves, especially since it makes the implementation more confusing to read. 

Is there a reason why the call to `CompositePageContainer` can't be in `V1#bake` etc?